cmd/machine-config-operator/bootstrap.go

```diff
@@ -62,6 +63,8 @@ func init() {
 	bootstrapCmd.MarkFlagRequired("etcd-image")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.setupEtcdEnvImage, "setup-etcd-env-image", "", "Image for Setup Etcd Environment.")
 	bootstrapCmd.MarkFlagRequired("setup-etcd-env-image")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.infraImage, "infra-image", "", "Image for Infra Containers.")
```

Comment:
do we need a default value according to https://github.com/openshift/machine-config-operator/pull/471/files#diff-c15e2a465008fe4b86940c96d54a17eaR3?

Reply:
Yes - until the installer change lands.

Reply:
looks like, anyway, there's still a chicken and egg issue where the MC generated at bootstrap in CI are different than the ones we now create, causing e2e-* to just fail. I'm again unsure about the flow.